tests/ImageSharp.Benchmarks/Samplers/Resize.cs

```diff
@@ -94,26 +101,44 @@ protected override void ExecuteResizeOperation(IImageProcessingContext<Rgba32> c
-        // Frequency=2742192 Hz, Resolution=364.6718 ns, Timer=TSC
-        // .NET Core SDK=2.1.602
+        // Frequency=2742189 Hz, Resolution=364.6722 ns, Timer=TSC
+        // .NET Core SDK=2.2.202
         //   [Host] : .NET Core 2.1.9 (CoreCLR 4.6.27414.06, CoreFX 4.6.27415.01), 64bit RyuJIT
-        //   Clr    : .NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.7.3362.0
+        //   Clr    : .NET Framework 4.7.2 (CLR 4.0.30319.42000), 64bit RyuJIT-v4.7.3394.0
         //   Core   : .NET Core 2.1.9 (CoreCLR 4.6.27414.06, CoreFX 4.6.27415.01), 64bit RyuJIT
         // 
-        // IterationCount=3  LaunchCount=1  WarmupCount=3
+        // IterationCount=3  LaunchCount=1  WarmupCount=3  
         // 
-        //                                    Method |  Job | Runtime | SourceSize | DestSize |      Mean |     Error |    StdDev | Ratio | Gen 0/1k Op | Gen 1/1k Op | Gen 2/1k Op | Allocated Memory/Op |
-        // ----------------------------------------- |----- |-------- |----------- |--------- |----------:|----------:|----------:|------:|------------:|------------:|------------:|--------------------:|
-        //                             SystemDrawing |  Clr |     Clr |       3032 |      400 | 118.71 ms |  4.884 ms | 0.2677 ms |  1.00 |           - |           - |           - |              2048 B |
-        //  'ImageSharp, MaxDegreeOfParallelism = 1' |  Clr |     Clr |       3032 |      400 |  94.55 ms | 16.160 ms | 0.8858 ms |  0.80 |           - |           - |           - |             16384 B |
-        //                                           |      |         |            |          |           |           |           |       |             |             |             |                     |
-        //                             SystemDrawing | Core |    Core |       3032 |      400 | 118.38 ms |  2.814 ms | 0.1542 ms |  1.00 |           - |           - |           - |                96 B |
-        //  'ImageSharp, MaxDegreeOfParallelism = 1' | Core |    Core |       3032 |      400 |  90.28 ms |  4.679 ms | 0.2565 ms |  0.76 |           - |           - |           - |             15712 B |
+        //                                    Method |  Job | Runtime | SourceToDest |      Mean |     Error |    StdDev | Ratio | RatioSD | Gen 0/1k Op | Gen 1/1k Op | Gen 2/1k Op | Allocated Memory/Op |
+        // ----------------------------------------- |----- |-------- |------------- |----------:|----------:|----------:|------:|--------:|------------:|------------:|------------:|--------------------:|
+        //                             SystemDrawing |  Clr |     Clr |     3032-400 | 120.11 ms |  1.435 ms | 0.0786 ms |  1.00 |    0.00 |           - |           - |           - |              1638 B |
+        //  'ImageSharp, MaxDegreeOfParallelism = 1' |  Clr |     Clr |     3032-400 |  75.32 ms | 34.143 ms | 1.8715 ms |  0.63 |    0.02 |           - |           - |           - |             16384 B |
```

Comment:
.NET Framework is faster?

Reply:
I'm also surprised, but happens quite often when you go full unsafe, dropping all the span indexers. It is also backed by RyuJIT.